Update_user.css

```diff
@@ -20,36 +27,68 @@ body{
 
 nav{
 	width:100%;
-	background-color:white;
+	background-color:#0C59CF;
```

Comment:
I change the background for the nav to navy blue which is the color theme that I should use for my music database.